typescript/src/entities/user.ts

```diff
@@ -246,7 +250,15 @@ export class User extends Entity {
     return matchingEnrolledDomains[0].role!;
   };
 
+  public isDomainManager(domain: Domain) {
+    return DOMAIN_MANAGERS.includes(this.roleInDomain(domain));
+  }
+
   public isManagementTeam(domain: Domain) {
     return MANAGEMENT_TEAM.includes(this.roleInDomain(domain));
   }
+
+  public isNotClient(domain: Domain) {
+    return BUSINESS_ACCOUNTS.includes(this.roleInDomain(domain));
```

Comment:
i have to note that merchicrm cannot import this as of `develop` -- webpack or whatever is not setup to understand `includes`

Reply:
have to talk about @s3341458 about this.